src/Form/FieldCheckbox.js

```diff
@@ -10,91 +10,123 @@ export default class FieldCheckbox extends Util.mixin(BindMixin) {
     return ['handleChange'];
   }
 
-  componentDidMount() {
-    if (this.props.startValue === 'indeterminate') {
-      React.findDOMNode(this.refs.checkbox).indeterminate = true;
-    }
+  handleChange(name, modelState) {
+    let {props} = this;
```

Comment:
I believe the official name (supported by all the standards) is "Bracket Sandwich"